templates/breadcrumb--ec-corporate.tpl.php

```diff
@@ -14,8 +14,9 @@
       <ol class="ecl-breadcrumb__container">
         <?php foreach ($breadcrumb as $value): ?>
           <?php if ($i < $count - 1): ?>
-            <li class="ecl-breadcrumb__segment" data-ecl-breadcrumb-item="static" aria-hidden="false">
+            <?php if (!empty($value['content']) && !empty($value['url'])) : ?>
```

Comment:
You are removing the <li> element so it will brake the HTML list.
There is already a clean treatment in the "breadcrumb__ec_corporate.component.inc" file line 18 so maybe adapt the code according your issue.

I wrote a PR to resolve your issue, please take a look. Thank you